website/docs/r/cloud_backup_schedule.html.markdown

```diff
@@ -146,6 +146,15 @@ resource "mongodbatlas_cloud_backup_schedule" "test" {
 * `policy_item_daily` - (Optional) Daily policy item
 * `policy_item_weekly` - (Optional) Weekly policy item
 * `policy_item_monthly` - (Optional) Monthly policy item
+* `auto_export_enabled` - Flag that indicates whether automatic export of cloud backup snapshots to the AWS bucket is enabled. Value can be one of the following:
+
+    true - enables automatic export of cloud backup snapshots to the AWS bucket
+    false - disables automatic export of cloud backup snapshots to the AWS bucket (default)
+* `use_org_and_group_names_in_export_prefix` - Specify true to use organization and project names instead of organization and project UUIDs in the path for the metadata files that Atlas uploads to your S3 bucket after it finishes exporting the snapshots. To learn more about the metadata files that Atlas uploads, see Export Cloud Backup Snapshot.
```

Comment:
```suggestion
* `use_org_and_group_names_in_export_prefix` - Specify true to use organization and project names instead of organization and project UUIDs in the path for the metadata files that Atlas uploads to your S3 bucket after it finishes exporting the snapshots. To learn more about the metadata files that Atlas uploads, see [Export Cloud Backup Snapshot](https://www.mongodb.com/docs/atlas/backup/cloud-backup/export/#std-label-cloud-provider-snapshot-export).
```

Reply:
Fixed

Reply:
same as above, missing brackets.

Reply:
Added brackets